src/Statamic.php

```diff
@@ -346,4 +348,26 @@ public static function frontendRouteSegmentRegex()
 
         return $prefix.'.*';
     }
+
+    public static function antlersTag($name, $params = [])
```

Comment:
Calling this `tag` is fine.

Reply:
Yeah I was on fence about this, you fellas are right, `tag()` is better.